.github/workflows/actions/download/action.yml

```diff
@@ -64,6 +64,13 @@ runs:
         cd ${{ github.workspace }}
         tar cfz RACK/rack-box/files/rack-cli.tar.gz RACK/cli/{*.sh,wheels}
 
+    - name: Package RACK UI
+      shell: bash
+      run: |
+        cd RACK/rack-ui
+        cd ${{ github.workspace }}
```

Comment:
The first cd command on line 70 is redundant and can be removed because the second cd command on line 71 changes the directory again.